web/content/docs/benchmarks/elliptic/elliptic-dirichlet.pandoc

```diff
@@ -102,21 +105,4 @@ A major part of the output was produced by the linear equation solver (LIS in th
 
 ![](../square_1e2_pcs_0_ts_1_t_1.000000.png)
 
-The result, written in the `square_1e2_result.dat`, can be visualized with Paraview, for example, by including the values into the `square_1x1_quad_1e2.vtu` file in the `PointData` section:
-```
-...
-
-
-1
-0.8
-0.6
-0.4
-0.2
-...
--0.6
--0.8
--1
-
-
-```
-(The next releases shall simplify this procedure.)
+The result can be visualized with Paraview.
```

Comment:
Reads easier in my opinion, if this is moved before the image:
```
The result can be visualized with Paraview:
 __
|  |
 --
```

Reply:
Move text before image.